index.sh

```diff
@@ -81,19 +103,21 @@ done
 cat <<EOF_SSL >> /tmp/nginx.ssl.conf
 # default config (server_name _; makes this 'base' config)
 server {
-  listen 443 default ssl;
+  listen 443 default ssl;  
   server_name _;
 
   ssl_certificate_key $(realpath "$KEY");
   ssl_certificate $(realpath "$CERT");
 
   # These this next block of settings came directly from the SSLMate recommend nginx configuration
   # Recommended security settings from https://wiki.mozilla.org/Security/Server_Side_TLS
-  ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
-  ssl_ciphers 'ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:kEDH+AESGCM:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA256:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA:DHE-RSA-AES256-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:AES:CAMELLIA:DES-CBC3-SHA:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!PSK:!aECDH:!EDH-DSS-DES-CBC3-SHA:!EDH-RSA-DES-CBC3-SHA:!KRB5-DES-CBC3-SHA';
+  ssl_protocols $TLS_VERSIONS;
+  ssl_ciphers '$CIPHERS';
```

Comment:
there shouldn't be ' around this

Reply:
are they not supposed to be single quoted?